extensions/codeql-csharp-extensions/models/Azure.yml

```diff
@@ -0,0 +1,40 @@
+extensions:
+  - addsTo:
+      pack: codeql/csharp-queries
+      extensible: sinkModel
+    data:
+      # > Cosmos DB
+      # https://learn.microsoft.com/en-us/dotnet/api/microsoft.azure.cosmos.querydefinition?view=azure-dotnet
+      - ["Microsoft.Azure.Cosmos", "QueryDefinition", False, "QueryDefinition", "(String)", "", "Argument[0]", "sql", "manual"]
+      # https://learn.microsoft.com/en-us/dotnet/api/microsoft.azure.cosmos.cosmosclient?view=azure-dotnet
+      - ["Microsoft.Azure.Cosmos", "CosmosClient", False, "CosmosClient", "(String,AzureKeyCredential,CosmosClientOptions)", "", "Argument[0]", "url", "manual"]
+      - ["Microsoft.Azure.Cosmos", "CosmosClient", False, "CosmosClient", "(String,CosmosClientOptions)", "", "Argument[0]", "url", "manual"]
+      - ["Microsoft.Azure.Cosmos", "CosmosClient", False, "CosmosClient", "(String,String,CosmosClientOptions)", "", "Argument[0]", "url", "manual"]
+      - ["Microsoft.Azure.Cosmos", "CosmosClient", False, "CosmosClient", "(String,String,CosmosClientOptions)", "", "Argument[1]", "password", "manual"]
+      - ["Microsoft.Azure.Cosmos", "CosmosClient", False, "CosmosClient", "(String,TokenCredential,CosmosClientOptions)", "", "Argument[0]", "url", "manual"]
+
+      # > ServiceBus (Message Queue)
+        # Assumes a threat model that anything could be put on the ServiceBus
+      # https://learn.microsoft.com/en-us/dotnet/api/azure.messaging.servicebus.servicebusreceiver?view=azure-dotnet
+      - ["Azure.Messaging.ServiceBus", "ServiceBusReceiver", False, "ReceiveMessageAsync", "", "", "ReturnValue", "remote", "manual"]
+      - ["Azure.Messaging.ServiceBus", "ServiceBusReceiver", False, "ReceiveMessagesAsync", "", "", "ReturnValue", "remote", "manual"]
+
+      - ["Azure.Messaging.ServiceBus", "ServiceBusReceivedMessage", False, "Body", "", "", "", "remote", "manual"]
+
+      # Storage - Blob
+      # TODO: Need to do more testing
```

Comment:
What kind of testing still need to be done here?

Reply:
Well none of these models will be loaded as `file-create` isn't a kind we support